org.gnome.Crosswords.PuzzleSets.oedipus.json

```diff
@@ -0,0 +1,34 @@
+{
+	"app-id": "org.gnome.Crosswords.PuzzleSets.oedipus",
+	"runtime": "org.gnome.Crosswords",
+	"runtime-version": "stable",
+	"branch": "1",
+	"sdk": "org.gnome.Sdk//42",
+	"build-extension": true,
+	"separate-locales": false,
+	"appstream-compose": false,
+	"build-options": {
+		"prefix": "/app/extensions"
+	},
+	"modules": [
+                "python3-requirements.json",
+		{
+			"name": "puzzle-sets-oedipus",
+			"buildsystem": "meson",
+			"config-opts": [
+				"-Dflatpak=true"
+			],
+			"sources": [
+				{
+					"type": "dir",
```

Comment:
it will never find the source. You need to specify a url to download a tarball or a git repository.